docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.1'
 services:
   app:
-    image: node:18
+    image: node:18-slim@sha256:b175cd7f3358c399f7bcee9b1032b86b71b1afa4cfb4dd0db55d66f871475a3e
```

Comment:
hm, what's the reason behind this update?

Reply:
@jpetto was getting `image with reference node:18 was found but does not match the specified platform: wanted linux/amd64, actual: linux/arm64/v8` error